lib/stm32/l0/rcc.c

```diff
@@ -408,5 +540,50 @@ void rcc_set_hpre(uint32_t hpre)
 	RCC_CFGR = reg | (hpre << RCC_CFGR_HPRE_SHIFT);
 }
 
+/*---------------------------------------------------------------------------*/
+/** @brief Common clock setups.
+ *
+ * @note After a reset, the system uses MSI@2.1MHz.
+ */
+
+void rcc_clock_setup_in_hsi16_out_32mhz(void)
+{
```

Comment:
nope.  Please don't add more of this style.  It's insane maintennance, and everyone wants to keep adding their own versions.  I want to see everyone move towards the f4 style, and see also some notes on a old pull from esden on rcc overhaul.  Adding methods like this has just been a disaster for maintaining consistent code.